src/modules/demos.js

```diff
@@ -165,6 +170,14 @@ export function *watchGetDemoSession() {
   }
 }
 
+export function *watchEndDemoSession() {
+  while (true) {
+    yield take(END_DEMO_SESSION);
```

Comment:
@rvennam could you also delete the demo on the backend? There is a DELETE endpoint on the controller /demos/<string:guid> this will remove all data from the ERP